packages/transmute-services/contracts/Oracle.sol

```diff
@@ -4,12 +4,6 @@ import "../node_modules/transmute-framework/contracts/TransmuteFramework/UnsafeE
 
 contract Oracle is UnsafeEventStore {
```

Comment:
Obviously we want to make this use the safe white list version. Interested to test how usage of that works.

Reply:
Yeah, this has not yet been implemented but is soon to be addressed.